liferay-lima/liferay-portal
# LPS-165760 / LPS-162327 Allow to select display page and minor fixes

# What is this about?

This renames the navigation fragment, puts it in the "Menu" section, and adds the last piece we missed in display page support: configuring a display page template per content.

![image](https://user-images.githubusercontent.com/729897/196926619-690bd195-33fa-442a-84ed-3c120004fb43.png)

# How do I test this?

When editing an article, open the "Display Page Template" section in the left pane and configure a specific display page template. Once saved, try to access the article with its url (e.g. /k/folder/article-1). Make sure the display page is not marked as default.
